Improve documentation of the review process (#117)
eds-book/eds-book.github.io: book/publishing/about/notebooks-technologies.md

(pb-about-technologies)=

# Technologies in EDS Book Notebooks
This section indicates the main technologies for peer-reviewing and sharing EDS book notebooks.

## Review
The interaction between authors and reviewers in the notebook repository is facilitated through [ReviewNB](https://www.reviewnb.com/), a third-party plugin in GitHub for displaying and commenting Jupyter Notebooks. 

> Sharing and viewing the content is much easier and faster than with any other platform e.g. Binder or GitHub recent feature that renders Jupyter Notebook rich diffs on (see [here](https://blog.reviewnb.com/github-not-rendering-interactive-notebook-widgets/)). 

Reviewer: ```suggestion
Sharing and viewing the content is much easier and faster than with any other platform, thanks to a feature that renders Jupyter Notebook rich diffs on (see [here](https://blog.reviewnb.com/github-not-rendering-interactive-notebook-widgets/)). 
```
Or something else to shorten/clarify this sentence, because I"m not sure what is meant here sorry!

Author: Fixed [07f7e23](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/07f7e2329e188808e04edddbb30954ba0c9df468).

The inline review comments interface is useful and user-friendly. 

> While it supports major visualisation libraries and interactive widgets, it does not render bokeh-related plots.

Reviewer: Would everyone know what bokey-related plots are?

Author: Fixed in [0f0bd07](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/0f0bd0771380efedda9ed43b7ded681f9b361550).

- [Visual diff](https://uploads-ssl.webflow.com/5ba4ebe021cb91ae35dbf88c/5ba93ded243329a486dab26e_sl-code%2Bimage.png) for every commit or pull request containing Notebooks
- [Commenting](https://uploads-ssl.webflow.com/5ba4ebe021cb91ae35dbf88c/5c7d0095d99ee508018a9878_Screenshot%202019-03-04%20at%204.08.48%20PM.png) on the notebook diff to start discussion
- [Conversation threads](https://uploads-ssl.webflow.com/5ba4ebe021cb91ae35dbf88c/5c7eb42d7cef320c0133d1c6_threads-v2.png) to track all open discussions
- [Team conversations](https://blog.reviewnb.com/commenting-for-jupyter/) directly on Jupyter Notebooks

In EDS book, we use ReviewNB in the review step. 
Once Editors-in-chief or reviewers open a PR in the notebook repository, it will include a link to view changes on ReviewNB.

You can find further information of latest ReviewNB developments in https://blog.reviewnb.com/.

## Research objects and RoHub
Research objects (ROs) are living resources helping to organise and describe the inputs, materials, and methods used in a scientific experiment and obtained as a result and not only at the end when publishing the research outcomes{cite}`ttw-2022`.

> [RoHub](https://reliance.rohub.org/)`rohub-2018` is a Research Object management platform that enables researchers to collaboratively manage, share and preserve their research work (data, software, workflows, models, presentations, videos, articles, etc.){cite}`ttw-2022`. 

Reviewer 2: ```suggestion
[RoHub](https://reliance.rohub.org/){cite}`rohub-2018` is a Research Object management platform that enables researchers to collaboratively manage, share and preserve their research work (data, software, workflows, models, presentations, videos, articles, etc.){cite}`ttw-2022`. 
```

> [RoHub](https://reliance.rohub.org/){cite}`rohub-2018` is a Research Object management platform that enables researchers to collaboratively manage, share and preserve their research work (data, software, workflows, models, presentations, videos, articles, etc.){cite}`ttw-2022`. 

Reviewer: I don't think the reference here is rendering properly

Author: The reference renders now. It seems `{cite}` doesn't work next to a standard URL link. Fixed in [f8b9314](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/f8b9314af3b5c4587adbc1d281318abe03a530a6).
